call-home/src/common/constants.rs

```diff
@@ -62,3 +62,6 @@ pub(crate) fn call_home_frequency() -> std::time::Duration {
         })
         .unwrap()
 }
+
+/// Release version
+pub(crate) const RELEASE_VERSION: &str = "2.0.1";
```

Comment:
We should not hardcode this, we'll keep getting it wrong as we forget, please use the version_info crate from the control-plane dep.

Reply:
https://github.com/openebs/mayastor-dependencies/tree/develop/version-info

.. this one?

Reply:
here: https://github.com/openebs/mayastor-extensions/blob/efe832075f7f2fd770a0a58286f8cd8fbf212403/k8s/plugin/src/main.rs#L20